index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins:wght@300;400;500&display=swap" rel="stylesheet"> 
+    <link rel="stylesheet" href="CSS/Styles.css">
+    <title>My Personal Portfolio Project</title>
+    <link rel="icon" href="/favicon.ico" type="image/x-icon">
+</head>
+<body>
+    <header>
+        <div class="header">
+            <img class="logo" src="Images/image_geometry_1.png" alt="Top Geometric Figures">
```

Comment:
According to the evaluation guidelines, all of the decorative images should be background images, kindly fix this.

<img width="454" alt="Screen Shot 2022-05-25 at 17 20 49" src="https://user-images.githubusercontent.com/82426682/170378052-c3c6aff3-9b62-419f-b039-c2907d493cd8.png">